invenio_rdm_records/records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -1,402 +1,714 @@
+        "file_id": {
+          "description": "TODO - Identifier of the file. - same as for bucket",
+          "type": "string"
         }
-      },
-      "dates": {
-        "description": "Date or date interval.",
-        "minItems": 0,
-        "type": "array",
-        "items": {
-          "type": "object",
-          "additionalProperties": false,
-          "properties": {
-            "description": {
-              "description": "Description of the date or date interval e.g. 'Accepted' or 'Available' (CV).",
-              "type": "string"
-            },
-            "end": {
-              "description": "End date.",
-              "type": "string",
-              "format": "date-time"
-            },
-            "start": {
-              "description": "Start date.",
-              "type": "string",
-              "format": "date-time"
-            },
-            "type": {
-              "description": "Type of the date interval."
-            }
-          },
-          "required": ["type"]
+      }
+    },
+
+    "agent-user": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "user": {
+          "type": "integer"
         }
-      },
-      "descriptions": {
-        "description": "Description for record.",
-        "type": "array",
-        "uniqueItems": true,
-        "items": {
+      }
+    },
+
+    "agent": {
+      "description": "An agent.",
+      "oneOf": [
+        {"$ref": "#/definitions/agent-user"}
+      ]
+    }
+  },
+
+
+  "properties": {
+    "$schema": {
+      "description": "JSONSchema declaration.",
+      "type": "string"
+    },
+    "id": {
+      "description": "Persistent record identifier (alphanumeric).",
+      "type": "string"
+    },
+    "pid": {"$ref": "#/definitions/internal-pid"},
+
+    "conceptid": {
+      "description": "Persistent concept record identifier (alphanumeric).",
+      "type": "string"
+    },
+    "conceptpid": {"$ref": "#/definitions/internal-pid"},
+
+    "pids": {
+      "additionalProperties": {"$ref": "#/definitions/external-pid"}
+    },
+
+    "metadata": {
+      "type": "object",
+      "description": "Resource metadata.",
+      "additionalProperties": false,
+      "properties": {
+
+        "resource_type": {"$ref": "#/definitions/resource_type"},
+
+        "creators": {
+          "description": "Creators of the resource.",
+          "type": "array",
+          "items": {
             "type": "object",
+            "additionalProperties": false,
+            "required": ["name"],
             "properties": {
-                "description": {
-                  "description": "Description/abstract for record.",
-                  "type": "string"
-                },
-                "type": {
-                  "description": "Type of description.",
+                "name": {"type": "string"},
+                "type": {"$ref": "#/definitions/nameType"},
+                "given_name": {"type": "string"},
```

Comment:
this was changed to `name` in the UI, needs consensus. For deduplication and others, having them splitter might be better.

Reply:
Intended, Personal vs Organization.